etc/profile-m-z/rpcs3.profile

```diff
@@ -0,0 +1,57 @@
+# Firejail profile for RPCS3 emulator
+# Description: RPCS3 emulator
+# This file is overwritten after every install/update
+# Persistent local customizations
+include rpcs3.local 
+# Persistent global definitions
+include globals.local # might be a problem
+
+noblacklist ${HOME}/.config/rpcs3
+noblacklist ${HOME}/.cache/rpcs3
+blacklist /usr/libexec
+
+#include disable-common.inc # not working: libstdc++.so.6: version `GLIBCXX_3.4.29' not found
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+#include disable-programs.inc # crashes PPU compilation
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.cache/rpcs3
+mkdir ${HOME}/.config/rpcs3
+whitelist ${HOME}/.cache/rpcs3
+whitelist ${HOME}/.config/rpcs3
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+net none
+netfilter
+nodvd
+nogroups
+#noinput
+nonewprivs
+noroot
+notv
+nou2f
+#novideo
+protocol unix,netlink
+seccomp
```

Comment:
`seccomp.block-secondary`?

Reply:
What does block-secondary do?

Reply:
> **--seccomp.block-secondary**
>              Enable seccomp filter and filter system call architectures so that only the native architecture  is  allowed.  For
             example,  on  amd64, i386 and x32 system calls are blocked as well as changing the execution domain with personal‐
              ity(2) system call.

Reply:
Thanks!